sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -302,19 +302,6 @@ def repository_name(self):
         return self._repository_name
 
 
-class DownloadBlobResult(object):
```

Comment:
We can't just remove this - it's GA?

Reply:
none of the blob and manifest upload/download was GA-ed  yet, so it should be ok to remove

Reply:
omg really?!?
Oh that makes everything _so_ much simpler! Thank you for confirming!!

Reply:
Sorry, I think I confused Anna ...